sdk/tests/client/node_api/mod.rs

```diff
@@ -46,7 +46,7 @@ pub fn setup_secret_manager() -> SecretManager {
 }
 
 // Sends a transaction block to the node to test against it.
-async fn setup_transaction_block() -> (BlockId, TransactionId) {
+pub async fn setup_transaction_block() -> (BlockId, TransactionId) {
     let client = setup_client_with_node_health_ignored().await;
```

Comment:
Can we pass the client in instead of creating it twice?

Reply:
are you referring to this https://github.com/qrayven/iota-sdk/blob/b54e493c4dfcdab8b58e6b136b4f2a45c2b74278/sdk/tests/client/node_api/mod.rs#L101 which is crated twice,   or you want pass optional `Client`  in the definition of function?

Reply:
This fn creates a client but does not return it, so it must be created by the caller anyway.

Reply:
done